### PR TITLE
[DBX-89576] injest new 0781 flipper from BE

### PR DESCRIPTION
**Pull request by SamStuckey**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

### :warning: TeamSites :warning:
Examples of a TeamSite: https://va.gov/health and https://benefits.va.gov/benefits/. This scenario is also referred to as the "injected" header and footer. You can reach out in the `#sitewide-public-websites` Slack channel for questions.

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

- [surface a value provided by the backend flipper in this PR](https://github.com/department-of-veterans-affairs/vets-api/pull/18720) in the Form 526 Submission flow.
- This will not be used right now. It will be leveraged during our 0781 paper form sync

## Related issue(s)

- [Ticket 1](https://github.com/department-of-veterans-affairs/va.gov-team/issues/92013)
- [Ticket 2](https://github.com/department-of-veterans-affairs/va.gov-team/issues/92199)

**NOTE: this change accomplishes both goals**

## Testing done

-  click through testing in a local dev env leveraging the flipper configuration in the backend app
- Confirmed the following:
  - no errors
  - flipper logic is correctly ingested
  - value is successfully saved by the Save In Progress Form logic

## Screenshots

N/a

## What areas of the site does it impact?

This *will* be used to roll out our 0781 paper form sync. There is no currently active usecase

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated (see tickets)
- [x] Screenshot of the developed feature is added (n/a - no visible changes)
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed (n/a, no user facing changes)

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution (n/a)
- [ ] Feature/bug has a monitor built into Datadog or Grafana (n/a)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

**Review by kylesoskin — APPROVED** [review]
Looks good.

**Comment by SamStuckey** [comment]
Note to reviewers, [these spec failures have been determined to be non-blocking](https://dsva.slack.com/archives/C5HP4GN3F/p1727974580490819?thread_ts=1727802245.653169&cid=C5HP4GN3F), afaik.

Also the code climate complaints seem extraneous. Open to feedback if y'all think a refactor is truly necessary

File: src/applications/disability-benefits/all-claims/prefill-transformer.js [hunk 1]
```diff
@@ -143,12 +143,24 @@ export default function prefillTransformer(pages, formData, metadata) {
     return newData;
   };
 
+  const prefillSyncModern0781Flow = data => {
+    const newData = _.omit(['syncModern0781Flow™'], data);
```
Comment (freeheeling) [inline src/applications/disability-benefits/all-claims/prefill-transformer.js:147]: Odd attribute to trademark.

**Review by pennja — APPROVED** [review]
Looks good! 👍

**Review by harshilmoradia — APPROVED** [review]

**Pull request merged**